services/ui-src/src/components/export/ExportedEntityDetailsTable.tsx

```diff
@@ -19,6 +19,8 @@ import verbiage from "verbiage/pages/mlr/mlr-export";
 export const ExportedEntityDetailsTable = ({
   fields,
   entity,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Fixes `Warning: React does not recognize the 'showHintText' prop on a DOM element.` but isn't used anywhere in the component, so it causes a lint error.